### PR TITLE
Adding image for OGW buckets

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Review by lproven — COMMENTED** [review]
(In old text)

> The content pane occupies the main part of the dashboard's screen. 
> The dashboard home page ...

Should this be using entities?

**Review by lproven — CHANGES_REQUESTED** [review]
Line 230:
> automatically refresh themselves by default every 5 seconds

five seconds

**Comment by asettle** [comment]
> (In old text)
> 
> > The content pane occupies the main part of the dashboard's screen.
> > The dashboard home page ...
> 
> Should this be using entities?

The entity translates to "Ceph Dashboard" - which we don't need to say every time :)

File: xml/admin_gui_dashboard.xml [hunk 1]
```diff
@@ -2424,9 +2430,16 @@ Please consult the documentation on how to configure and enable the Object Gatew
...
      To add a new &ogw; bucket, click <guimenu>Create</guimenu> in the top left of
-     the table heading. Enter the bucket's name and select its owner. Confirm
-     with <guimenu>Create Bucket</guimenu>.
+     the table heading. Enter the bucket's name, select the user an owner, and
+     set the placement target. Confirm with <guimenu>Create Bucket</guimenu>.
     </para>
+    <note>
+      <para>
+        At this stage you can also enable locking by selecting <guimenu>Enabled</guimenu>,
+        however this is configurable after creation. See <xref linkend="dashboard-ogw-bucket-edit"/>
```
Comment (lproven) [inline xml/admin_gui_dashboard.xml:2439]:
```suggestion
        At this stage you can also enable locking by selecting <guimenu>Enabled</guimenu>;
        however, this is configurable after creation. See <xref linkend="dashboard-ogw-bucket-edit"/>
```

**Review by lproven — CHANGES_REQUESTED** [review]
Line 2459:
>  Select and highlight a bucket, click <guimenu>Edit</guimenu> in the top

I suggest:
 Select and highlight a bucket, then click <guimenu>Edit</guimenu> in the top

**Comment by lproven** [comment]
> The entity translates to "Ceph Dashboard" - which we don't need to say every time :)

OK! That's why I asked, not **suggest**ed...

**Comment by asettle** [comment]
[book-storage-admin_color_en.pdf](https://github.com/SUSE/doc-ses/files/5136869/book-storage-admin_color_en.pdf)

File: xml/admin_gui_dashboard.xml [hunk 2]
```diff
@@ -2424,9 +2430,16 @@ Please consult the documentation on how to configure and enable the Object Gatew
     <title>Adding a New Bucket</title>
     <para>
      To add a new &ogw; bucket, click <guimenu>Create</guimenu> in the top left of
-     the table heading. Enter the bucket's name and select its owner. Confirm
-     with <guimenu>Create Bucket</guimenu>.
+     the table heading. Enter the bucket's name, select the user an owner, and
```
Comment (lproven) [inline xml/admin_gui_dashboard.xml:2433]:
```suggestion
     the table heading. Enter the bucket's name, select the user and owner, and
```

Comment (lproven) [reply xml/admin_gui_dashboard.xml:2433]: I am guessing you meant "and" but ISTM it could have been "or"...

**Review by lproven — CHANGES_REQUESTED** [review]

**Review by lproven — APPROVED** [review]

File: xml/admin_gui_dashboard.xml (hunk 2 above)
Comment (asettle) [reply xml/admin_gui_dashboard.xml:2433]: oh that's an odd one. I didn't mean either. Essentially, you seelct the user *for* the ownership (of the bucket). But in the gui, you jsut select a user for the owner. See: 
Screenshot from 2020-08-27 16-45-39

Thoughts?

Comment (lproven) [reply xml/admin_gui_dashboard.xml:2433]: How about just "select the owner"?

File: xml/admin_gui_dashboard.xml [hunk 3]
```diff
@@ -2463,6 +2476,17 @@ Please consult the documentation on how to configure and enable the Object Gatew
      You can update the owner of the bucket or enable versioning, multi-factor
      authentication or locking. Confirm any changes with <guimenu>Edit Bucket</guimenu>.
     </para>
+    <figure>
+     <title>Editing the Gateway Bucket Details</title>
```
Comment (LenzGr) [inline xml/admin_gui_dashboard.xml:2480]: "Gateway Bucket Details" sounds weird. I'd remove the "Gateway" part.

**Review by LenzGr — APPROVED** [review]
LGTM otherwise, thanks!

File: xml/admin_gui_dashboard.xml [hunk 4]
```diff
@@ -207,6 +207,12 @@
       <para>
        View and manage &ogw;'s daemons, users, and buckets.
       </para>
+      <note>
+        <para>
+          If &ogw; is not deployed, an information notice appears.
```
Comment (lproven) [inline xml/admin_gui_dashboard.xml:212]: "Informative notice"?

**Review by lproven — COMMENTED** [review]
_Pace_ my outstanding query from last night, and the one new query, otherwise fine.

**Pull request merged**